app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
 
     <!-- To be able to install APK from the application -->
     <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES" />
-
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
```

Comment:
This declaration should be located in the AndroidManifest of the module where `FloatingVideoService` is defined.